metricbeat/module/windows/perfmon/perfmon.go

```diff
@@ -40,13 +40,18 @@ type CounterConfig struct {
 
 // Config for the windows perfmon metricset.
 type Config struct {
-	IgnoreNECounters  bool            `config:"perfmon.ignore_non_existent_counters"`
-	GroupMeasurements bool            `config:"perfmon.group_measurements_by_instance"`
-	CounterConfig     []CounterConfig `config:"perfmon.counters" validate:"required"`
+	IgnoreNECounters   bool            `config:"perfmon.ignore_non_existent_counters"`
+	GroupMeasurements  bool            `config:"perfmon.group_measurements_by_instance"`
+	CounterConfig      []CounterConfig `config:"perfmon.counters" validate:"required"`
+	GroupAllCountersTo string          `config:"perfmon.group_all_counter"` // only available for the iis/webserver metricset at the moment
+	// could be used in the future if any solutions are built for https://github.com/elastic/beats/issues/16366
+	//FilterByInstance   []string        `config:"perfmon.filter_by_instance"` // only available for the iis/webserver metricset at the moment
 }
 
+const metricsetName = "perfmon"
+
 func init() {
-	mb.Registry.MustAddMetricSet("windows", "perfmon", New)
+	mb.Registry.MustAddMetricSet("windows", metricsetName, New)
```

Comment:
I am thinking now that as these metricsets don't need a host because they only query perfmon, we could add this:
```suggestion
	mb.Registry.MustAddMetricSet("windows", metricsetName, New,
	                    mb.WithHostParser(parse.EmptyHostParser),
	)
```